BrainPortal/public/javascripts/cbrain.js

```diff
@@ -239,59 +230,27 @@
         project_details.hide();
```

Comment:
This corresponds to: `.project_button .project_button_details { display: none; }`